api/v1/flow/flow.proto

```diff
@@ -472,3 +472,91 @@ message LostEvent {
     // PERF_EVENT_RING_BUFFER.
     google.protobuf.Int32Value cpu = 3;
 }
+
+// AgentEventType is the type of agent event. These values are shared with type
+// AgentNotification in pkg/monitor/api/types.go
+enum AgentEventType {
+    UNSPECIFIED = 0;
+    START = 1;
```

Comment:
nit: maybe call this one `AGENT_STARTED`

Again, I know this is copied over from `pkg/monitor/api/types.go`, but in protobuf we don't have enum namespacing and so no other enum can ever call a variant `START` if we do this.

Reply:
Agree, changing to `AGENT_STARTED`.